draft-ietf-quic-qpack.md

```diff
@@ -1339,7 +1341,7 @@ An attacker can try to cause an endpoint to exhaust its memory. QPACK is
 designed to limit both the peak and stable amounts of memory allocated by an
 endpoint.
 
-The amount of memory used by the encoder is limited by the protocol using
+The amount of memory used by the decoder is limited by the protocol using
```

Comment:
In some sense this is true as it is, but that's not a terribly interesting statement to make -- the encoder can limit its memory usage unilaterally, and could be sloppily implemented and exceed the protocol-conveyed limit.